articles/native-interop/facebook-login.md

```diff
@@ -278,7 +279,7 @@ to store any state in the surrounding Uno object.
 
 The login method is implemented using foreign Objective-C as follows:
 
-```
+```csharp
 [Foreign(Language.ObjC)]
 public extern(iOS) void Login(Action<AccessToken> onSuccess, Action onCancelled, Action<string> onError)
 @{
```

Comment:
Hmm, I have a feeling that this isn't giung to render very nicely. Foreign code is a bit odd to syntax-highlight... What does the result look like?

Reply:
Its alright, something like this: https://github.com/fuse-open/docs/blob/6d87df85b6cbfd64d0d6c57a04a6a205174d7aaa/articles/native-interop/facebook-login.md

Reply:
Yep, you're right! This renders nicely!